db/schema.rb

```diff
@@ -0,0 +1,40 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2021_12_27_053009) do
+
+  create_table "comments", force: :cascade do |t|
+    t.text "content"
```

Comment:
Here are the actual attributes you need to create a comment:
Also, the comment needs a `post id`, so the command on the console will be e.g. ⬇️ 
`@post = Post.find_by(id: 5)
@post.comments.create(content: "my comment", name: "Jasper Jan")
//or you can assign the post_id upon creation --> Comment.create(post_id: 5, content: "my comment", name: "Jasper Jan") `